promptmeteo/base.py

```diff
@@ -62,6 +63,7 @@ def __init__(
         prompt_detail: Optional[str] = None,
         selector_k: int = 10,
         selector_algorithm: str = "relevance",
+        selector_k_per_class: Optional[int] = 2,
```

Comment:
I'm not sure that this is the better way to do this, but i see that you need it to call `build_selector_by_load`.

Let's go with that change, but we must think in a better way to extend the base class

Reply:
Hi! Do you mean to have 'selector_k_per_class' as object variable?

Reply:
This is solved :)